examples/devenv-django/devenv.nix

```diff
@@ -0,0 +1,87 @@
+{ pkgs, ... }:
+
+let
+  db_user = "postgres";
+  db_host = "localhost";
+  db_port = "5432";
+  db_name = "db";
+  python_version = "3.10";
+in
+{
+  packages = [ pkgs.git pkgs.postgresql_14 ];
+
+  languages.python = {
+    enable = true;
+    poetry.enable = true;
+  };
+
+  env = {
+    PYTHON_VERSION = python_version;
+    DATABASE_URL = "postgresql://${db_user}@${db_host}:${db_port}/${db_name}";
+    DEBUG = true;
+    STATIC_ROOT = "/tmp/static";
+  };
+
+  enterShell = ''
+    # Activate poetry environment on shell entry
+    # https://pythonspeed.com/articles/activate-virtualenv-dockerfile/
+    VIRTUAL_ENV=`poetry env info --path` && export VIRTUAL_ENV 
+    export PATH="$VIRTUAL_ENV/bin:$PATH";
```

Comment:
This should already have been set by the devenv python module. Was this actually needed?

Reply:
Nope,  this override is from before adding `poetry.enable = true` - `pip list` gives me the same result with or without it 😉 

I'll remove it